lib/Runtime/Library/IntlEngineInterfaceExtensionObject.cpp

```diff
@@ -1228,164 +1237,201 @@ namespace Js
 #endif
     }
 
-    DWORD getFlagsForSensitivity(LPCWSTR sensitivity)
+#ifdef INTL_WINGLOB
+    static DWORD GetCompareStringComparisonFlags(CollatorSensitivity sensitivity, bool ignorePunctuation, bool numeric)
```

Comment:
> static DWORD GetCompareStringComparisonFlags(CollatorSensitivity sensitivity, bool ignorePunctuation, bool numeric) [](start = 3, length = 116)

Looks like the change here is to change from strings to an enum and to add support for `ignorePunctuation` and `numeric` -- is that correct?

Reply:
yeah, it centralizes all of the flag manipulation that used to happen in the caller so as to minimize the size of the #ifdef'd blocks